lib/manageiq/api/client/action.rb

```diff
@@ -0,0 +1,17 @@
+module ManageIQ
+  module API
+    module Client
+      class Action
+        attr_accessor :name
+        attr_accessor :method
+        attr_accessor :href
```

Comment:
I personally, prefer just inlining these 3, but if you are going to add documentation, then this is fine.